tasks/extract_filepaths_for_granule/extract_filepaths_for_granule.py

```diff
@@ -52,13 +66,18 @@ def task(event, context):    #pylint: disable-msg=unused-argument
                 level = "event['input']['granules'][]['files']"
                 file_name = os.path.basename(afile['fileName']) if \
                     re.search('^s3://', afile['fileName']) else afile['fileName']
-                fkey = afile['key']
-                dest_bucket = None
-                for key in regex_buckets:
-                    pat = re.compile(key)
-                    if pat.match(file_name):
-                        dest_bucket = regex_buckets[key]
-                files.append({'key': fkey, 'dest_bucket': dest_bucket})
+                LOGGER.info(f"grabbed filename {file_name}")
+                # filtering excludeFileTypes
+                if not should_exclude_files_type(file_name, exclude_file_types):
+                    fkey = afile['key']
+                    LOGGER.info(f"grabbed file key {fkey}")
+                    dest_bucket = None
+                    for key in regex_buckets:
+                        pat = re.compile(key)
+                        if pat.match(file_name):
+                            dest_bucket = regex_buckets[key]
+                    files.append({'key': fkey, 'dest_bucket': dest_bucket})
+                    LOGGER.info(f"added to files- 'key': {fkey}, 'dest_bucket': {dest_bucket}")
```

Comment:
I would probably remove this log message, not really needed as we could pull the information from the step function result.